controllers/src/test/scala/cromwell/pipeline/controller/ProjectConfigurationControllerTest.scala

```diff
@@ -67,5 +67,29 @@ class ProjectConfigurationControllerTest extends AsyncWordSpec with Matchers wit
         }
       }
     }
+
+    "deactivate configuration" should {
+
+      "return success for deactivation configuration" in {
+        when(configurationService.deactivateConfiguration(configuration.projectId))
+          .thenReturn(Future.successful("Success"))
+        Delete("/configurations?project_id=" + configuration.projectId.value) ~> configurationController.route(
+          accessToken
+        ) ~> check {
+          status shouldBe StatusCodes.OK
+          entityAs[String] shouldBe "Success"
+        }
+      }
+
+      "return InternalServerError when failure deactivation configuration" in {
+        when(configurationService.deactivateConfiguration(configuration.projectId)).thenReturn(Future.failed(error))
+        Delete("/configurations?project_id=" + configuration.projectId.value) ~> configurationController.route(
+          accessToken
+        ) ~> check {
+          status shouldBe StatusCodes.InternalServerError
+          entityAs[String] shouldBe "Something went wrong"
```

Comment:
This sounds very abstract))  force your imagination and logic sense!) If I saw such an error I would feel scared)

Reply:
I think, that we can leave as it is, because in this test we just test that the general error in service would be with expected behavior